### ISSUE TITLE
Change log is missing from readme.txt, the separate changelog file is not updated

### ISSUE THREAD

**Issue by vlood** [auth0/wp-auth0#346]
As this is a WordPress plugin, it would be pretty appropriate to have a maintained change log in readme.txt. Dunno why it's gone now, but the lack of it brings way too much pain into poor WordPress developers than the amounts they signed up for.

**Comment by joshcanhelp** [comment]
@vlood - Agreed. I'll work on putting this together.

**Comment by joshcanhelp** [comment]
I've got a start here:

https://wordpress.org/plugins/auth0/#developers

... and will work adding releases there as well. We've got a more complete one here:

https://github.com/auth0/wp-auth0/blob/master/CHANGELOG.md

... and we're discussing internally where we want to maintain that instead of in both places. For now I'll close this since it does exist in the plugin now. 

Thanks for the report here 👍
